catalyst/dl/experiment/config.py

```diff
@@ -535,6 +542,14 @@ def get_callbacks(self, stage: str) -> "OrderedDict[Callback]":
             if not is_already_present:
                 callbacks[callback_name] = callback_fn()
 
+        # Remove LoggerCallback on worker nodes
+        if get_rank() > 0:
+            to_del = (LoggerCallback, ConfusionMatrixCallback)
+            for k in filter(
+                    lambda c: isinstance(callbacks[c], to_del), callbacks
+            ):
+                del callbacks[k]
+
```

Comment:
check CI errors please:
```
[07:51:07][Step 1/1] Traceback (most recent call last):
[07:51:07][Step 1/1]   File "catalyst/dl/scripts/run.py", line 114, in <module>
[07:51:07][Step 1/1]     main(args, unknown_args)
[07:51:07][Step 1/1]   File "catalyst/dl/scripts/run.py", line 109, in main
[07:51:07][Step 1/1]     distributed_run(args.data_parallel, main_worker, args, unknown_args)
[07:51:07][Step 1/1]   File "/opt/buildagent/work/b2d9bf5b17af067d/catalyst/utils/distributed.py", line 130, in distributed_run
[07:51:07][Step 1/1]     worker_fn(*args, **kwargs)
[07:51:07][Step 1/1]   File "catalyst/dl/scripts/run.py", line 104, in main_worker
[07:51:07][Step 1/1]     runner.run_experiment(experiment, check=check_run)
[07:51:07][Step 1/1]   File "/opt/buildagent/work/b2d9bf5b17af067d/catalyst/core/runner.py", line 379, in run_experiment
[07:51:07][Step 1/1]     raise ex
[07:51:07][Step 1/1]   File "/opt/buildagent/work/b2d9bf5b17af067d/catalyst/core/runner.py", line 374, in run_experiment
[07:51:07][Step 1/1]     self._run_stage(stage)
[07:51:07][Step 1/1]   File "/opt/buildagent/work/b2d9bf5b17af067d/catalyst/core/runner.py", line 336, in _run_stage
[07:51:07][Step 1/1]     self._prepare_for_stage(stage)
[07:51:07][Step 1/1]   File "/opt/buildagent/work/b2d9bf5b17af067d/catalyst/dl/core/runner.py", line 17, in _prepare_for_stage
[07:51:07][Step 1/1]     super()._prepare_for_stage(stage=stage)
[07:51:07][Step 1/1]   File "/opt/buildagent/work/b2d9bf5b17af067d/catalyst/core/runner.py", line 182, in _prepare_for_stage
[07:51:07][Step 1/1]     callbacks = self.experiment.get_callbacks(stage)
[07:51:07][Step 1/1]   File "/opt/buildagent/work/b2d9bf5b17af067d/catalyst/dl/experiment/config.py", line 549, in get_callbacks
[07:51:07][Step 1/1]     lambda c: isinstance(callbacks[c], to_del), callbacks
[07:51:07][Step 1/1] RuntimeError: OrderedDict mutated during iteration
```

Reply:
@AndreySheka

Reply:
Needs to revert commit https://github.com/catalyst-team/catalyst/pull/639/commits/129980e53f92233012e7f85079957f865dfe056e or just wrap `filter` by `list`

Reply:
@Scitator I fixed bugs in previous PR. You can make rebase.